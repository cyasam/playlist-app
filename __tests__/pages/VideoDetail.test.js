import React from 'react'
import { shallow } from 'enzyme'
import VideoDetail from '../../src/scripts/pages/VideoDetailpage'

describe('Video Detail Page', () => {
  const wrapper = shallow(<VideoDetail />)

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})