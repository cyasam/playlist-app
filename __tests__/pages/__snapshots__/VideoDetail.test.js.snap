// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video Detail Page renders properly 1`] = `
<Route
  render={[Function]}
/>
`;
